api-graphql/graph/model/requirement.go

```diff
@@ -0,0 +1,91 @@
+package model
+
+type Requirement interface {
```

Comment:
I would suggest maybe 'Meets()' method that is some way you can show that the requirement has been met?